scripts/import_default_vpc.py

```diff
@@ -0,0 +1,79 @@
+from more_itertools import (
+    one,
+)
+
+from azul import (
+    config,
+    logging,
+)
+from azul.deployment import (
+    aws,
+)
+from azul.logging import (
+    configure_script_logging,
+)
+from azul.terraform import (
+    terraform,
+)
+
+log = logging.getLogger(__name__)
+
+
+def main() -> None:
+    if config.terraform_component == 'shared':
+        import_default_vpc()
+    else:
+        log.info('The "shared" component is not selected, skipping import.')
+
+
+def get_default_vpc_id() -> str:
+    filters = [
+        {
+            'Name': 'isDefault',
+            'Values': ['true']
+        }
+    ]
+    vpcs = aws.ec2.describe_vpcs(Filters=filters)
+    return one(vpcs['Vpcs'])['VpcId']
+
+
+def get_default_security_group_id(vpc_id: str) -> str:
+    filters = [
+        {
+            'Name': 'vpc-id',
+            'Values': [vpc_id]
+        },
+        {
+            'Name': 'group-name',
+            'Values': ['default']
+        }
+    ]
+    groups = aws.ec2.describe_security_groups(Filters=filters)
+    return one(groups['SecurityGroups'])['GroupId']
+
+
+def import_default_vpc() -> None:
+    resources = terraform.run('state', 'list').splitlines()
+
+    vpc_id = None
+
+    if 'aws_default_vpc.default' in resources:
+        log.info('The default VPC has already been imported.')
+    else:
+        vpc_id = get_default_vpc_id()
+        log.info('Importing the default VPC.')
+        terraform.run('import', 'aws_default_vpc.default', vpc_id)
+
+    if 'aws_default_security_group.default' in resources:
+        log.info("The default VPC's default security group has already been imported.")
+    else:
+        if vpc_id is None:
+            vpc_id = get_default_vpc_id()
```

Comment:
I'd prefer not to do this twice. I also prefer not to do this unnecessarily. It's possible to achieve both.